src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/ObjectWriter/CodeView/CodeViewSymbolsBuilder.cs

```diff
@@ -238,7 +238,7 @@ private CodeViewRegister GetCVRegNum(uint regNum)
 
                 foreach (var sequencePoint in sequencePoints)
                 {
-                    if (lastFileName == null || lastFileName != sequencePoint.FileName)
+                    if (lastFileName is null || lastFileName != sequencePoint.FileName)
```

Comment:
If you prefer pattern matching terse syntax, we can reduce the verbosity a bit in those long RelocType.XX conditions:

```c#
using static ILCompiler.DependencyAnalysis.RelocType;
...
if (relocType is IMAGE_REL_BASED_ARM64_BRANCH26 or IMAGE_REL_BASED_ARM64_PAGEBASE_REL21 or
     IMAGE_REL_BASED_ARM64_PAGEOFFSET_12A or IMAGE_REL_AARCH64_TLSLE_ADD_TPREL_HI12 or ..)
...
```
(the duplication due to `IMAGE_REL_` prefix is enough as is 😅)

Reply:
I used several different code styles as I progressed, sometimes intentionally, sometimes as a consequence of reusing existing code... I am generally open to ideas how to keep the code as terse and readable as possible ;-)